docs/guides/admin/docs/configuration/https/migration.md

```diff
@@ -0,0 +1,45 @@
+Migrating old content to HTTPS
```

Comment:
Note that this is an old guide. I just moved this to a separate documentation page.